### PR TITLE
OSSM-4535: Service Mesh 2.4.2 Z Stream Release Notes

### PR DESCRIPTION
**Pull request by rh-tokeefe**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OSSM-4535

Link to docs preview: https://62693--docspreview.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-version-2-4-2

QE review:
- [x] QE has approved this change.

Additional information:

Aligned team Doc Devtools
Dev Review jewertow
QA Review pbajjuri20
Peer review bburt-rh

### CODE REVIEW

File: modules/ossm-rn-known-issues.adoc [hunk 1]
```diff
@@ -36,6 +36,29 @@ endif::openshift-rosa[]
...
+  name: disable-istio-mtls-for-tls-app
+  namespace: default
+spec:
+  celector: 
+    matchLabels:
+      …
+  mtls:
+    mode: DISABLE
+----
+
```
Comment (jewertow) [inline modules/ossm-rn-known-issues.adoc:61]: We are postponing this bug fix to 2.4.3, so please remove it.
```suggestion
```

**Review by jewertow — CHANGES_REQUESTED** [review]

File: modules/ossm-rn-known-issues.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-rn-known-issues.adoc]: Removed this known issue based on discussion with Jacek.

**Review by jewertow — APPROVED** [review]

**Review by jewertow — COMMENTED** [review]
LGTM

File: modules/ossm-rn-new-features.adoc [hunk 2]
```diff
@@ -168,6 +190,28 @@ spec:
...
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.10 and later versions.
+
+=== Component versions included in {SMProductName} version 2.3.6
+
+|===
+|Component |Version
+
+|Istio
+|1.14.6
```
Comment (pbajjuri20) [inline modules/ossm-rn-new-features.adoc:203]: oc rsh -n istio-system23 istio-egressgateway-7d7cb9d55b-4gcl7 env | grep ISTIO_VERSION 
ISTIO_VERSION=1.14.5

It is 1.14.5

**Review by pbajjuri20 — APPROVED** [review]

File: modules/ossm-rn-new-features.adoc [hunk 3]
```diff
@@ -15,6 +15,28 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+== New features {SMProductName} version 2.4.2
```
Comment (bburt-rh) [inline modules/ossm-rn-new-features.adoc:18]: - should `2.4.2` use the `SMProductVersion` attribute that is defined in the `common-atributes.adoc` file?

File: modules/ossm-rn-new-features.adoc [hunk 4]
```diff
@@ -15,6 +15,28 @@ Module included in the following assemblies:
...
+
+This release of {SMProductName} addresses Common Vulnerabilities and Exposures (CVEs), contains bug fixes, and is supported on {product-title} 4.10 and later versions.
+
+=== Component versions included in {SMProductName} version 2.4.2
```
[the remark this answers is not in the document]
Comment (rh-tokeefe) [reply modules/ossm-rn-new-features.adoc:22]: We stopped using an attribute for the version in places where it didn't add value. All we were doing was using an attribute in the section title for the initial release, then in the next release changing it to static text, and using the attribute in the latest release. It was extra work for no genuine value add in the titles. There are a handful of places in the doc where it does add value and we continue to use the attributes in contexts where it makes sense.

**Comment by bburt-rh** [comment]
lgtm

**Comment by gwynnemonahan** [comment]
Merge review request made as 2.4.2 has been released. Tim O'Keefe is on PTO and i am covering for him.

**Comment by bburt-rh** [comment]
Merged into main and CP'ed into the 4.10+ enterprise branches.

**Pull request merged**